onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/DmlOperatorElementWise.cpp

```diff
@@ -558,7 +558,9 @@ class DmlOperatorElementwiseQLinear : public DmlOperator
             {
                 ML_CHECK_VALID_ARGUMENT(axis < outputShapeDimCount);
                 uint32_t broadcastAxisLength = outputShape[axis];
-                ML_CHECK_VALID_ARGUMENT(inputTensorShape[0] == broadcastAxisLength);
+                ML_CHECK_VALID_ARGUMENT((inputTensorShape[0] == broadcastAxisLength) ||
+                                        // treat as broadcast dimension to match CPU behavior
+                                        (inputTensorShape[0] == 1)); 
```

Comment:
[nit] Consistent indentation with rest of DML EP (uniformly 4 spaces per indentation level):

```
                ML_CHECK_VALID_ARGUMENT(
                    (inputTensorShape[0] == broadcastAxisLength) ||
                    // treat as broadcast dimension to match CPU behavior
                    (inputTensorShape[0] == 1)
                ); 
```